source/simulator/core.cc

```diff
@@ -109,12 +109,8 @@ namespace aspect
       if (geometry_model.has_curved_elements())
         return std::make_unique<MappingQCache<dim>>(4);
 
-#if !DEAL_II_VERSION_GTE(9,4,0) || DEAL_II_VERSION_GTE(9,4,1)
```

Comment:
Just talking myself through this: The condition means that deal.II either (i) is prior to 9.4, or (ii) 9.4.1 or later. The former cannot happen any more, but it is conceivable that someone uses deal.II 9.4.0. So I think the right change here is to replace this line by
```
#if DEAL_II_VERSION_GTE(9,4,1)
```
and leave the rest of the code we have here.

Reply:
Oh right, looked over the '!'